### ISSUE TITLE
Sampling rate conversion

### ISSUE THREAD

**Issue by bsshruthi22** [lhotse-speech/lhotse#1229]
I have mixed wav files of sampling rates 8khz and 16khz. When I try to import kaldi files for creating manifests, for eg:
lhotse kaldi import data/train 8000 train_manifests ,I get errors due to different sample rates (for 16khz files). Is there any way I can use them without manually converting all 16khz to 8khz?
The pipe command in wav.scp to convert to 8khz doesnt work and throws error when creating manifests.
Please suggest.

**Comment by bsshruthi22** [comment]
@pzelasko Thank you for the reply.I have prepared the manifests seperately and merged them.Is it possible to train with 2 different sampling frequencies together (8khz and 16khz)?

**Comment by pzelasko** [comment]
Yes. As long as Lhotse knows the actual sampling rate it will work. Typically you’d want to resample on the fly or before feature extraction with cuts.resample(16000).
